ChatQnA/README_miscellaneous.md

```diff
@@ -0,0 +1,85 @@
+# Table of contents
+
+1. [Build MegaService Docker Image](#Build-MegaService-Docker-Image)
+2. [Build UI Docker Image](#Build-UI-Docker-Image)
+3. [Build Conversational React UI Docker Image](#Build-Conversational-React-UI-Docker-Imaged)
```

Comment:
`(#Build-Conversational-React-UI-Docker-Imaged)` should be `(#Build-Conversational-React-UI-Docker-Image)`.